### PR TITLE
Task-50286 : Automatic Translation - Add 3 dots menu for activities and comments

### PR DESCRIPTION
**Pull request by rdenarie**
This commit add the possibility to call translationService to translate an activity or a comment

### CODE REVIEW

File: services/api/src/main/java/org/exoplatform/automatic/translation/api/AutomaticTranslationService.java [hunk 1]
```diff
@@ -27,4 +27,5 @@
   boolean setActiveConnector(String name);
   boolean setApiKey(String connector, String apikey);
   String translate(String message, Locale targetLang);
+  boolean isActiveFeature();
```
Comment (plamarque) [inline services/api/src/main/java/org/exoplatform/automatic/translation/api/AutomaticTranslationService.java]:
```suggestion
  boolean isFeatureActive();
```

File: services/impl/src/main/java/org/exoplatform/automatic/translation/impl/AutomaticTranslationServiceImpl.java [hunk 2]
```diff
@@ -63,26 +63,27 @@ public void addConnector(AutomaticTranslationComponentPlugin translationConnecto
...
 
-
-    return value != null && value.getValue()!=null && !value.getValue().toString().isEmpty() && isActiveFeature() ?
+    return value != null && value.getValue()!=null && !value.getValue().toString().isEmpty() && translationConnectors.get(value.getValue().toString())!=null ?
            value.getValue().toString() : null;
   }
 
 
-  private boolean isActiveFeature() {
+  public boolean isActiveFeature() {
```
Comment (plamarque) [inline services/impl/src/main/java/org/exoplatform/automatic/translation/impl/AutomaticTranslationServiceImpl.java]:
```suggestion
  public boolean isFeatureActive() {
```

File: webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2020 eXo Platform SAS.
+    
+    This is free software; you can redistribute it and/or modify it
+    under the terms of the GNU Lesser General Public License as
```
Comment (plamarque) [inline webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml]: AGPL

File: webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2020 eXo Platform SAS.
```
Comment (plamarque) [inline webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml]: 2021

**Review by plamarque — CHANGES_REQUESTED** [review]
make sure license header is AGPL and copyright 2021

File: services/api/src/main/java/org/exoplatform/automatic/translation/api/AutomaticTranslationService.java (hunk 1 above)
Comment (rdenarie) [reply services/api/src/main/java/org/exoplatform/automatic/translation/api/AutomaticTranslationService.java]: changed

File: services/impl/src/main/java/org/exoplatform/automatic/translation/impl/AutomaticTranslationServiceImpl.java (hunk 2 above)
Comment (rdenarie) [reply services/impl/src/main/java/org/exoplatform/automatic/translation/impl/AutomaticTranslationServiceImpl.java]: changed

File: webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml (hunk 4 above)
Comment (rdenarie) [reply webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml]: changed

File: webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml (hunk 3 above)
Comment (rdenarie) [reply webapps/src/main/webapp/WEB-INF/conf/automatic-translation/dynamic-container-configuration.xml]: changed

**Review by plamarque — APPROVED** [review]
license headers OK now.

**Pull request merged**